### PR TITLE
Correspondence of CodeIgniter4 #1108

### PR DESCRIPTION
**Pull request by ytetsuro**
Removed processing to change rewrite.php.

### CODE REVIEW

**Comment by kenjis** [comment]
I want to keep the command `bin/server.sh`.

**Comment by ytetsuro** [comment]
> I want to keep the command bin/server.sh.

OK! Fixed!

File: README.md [hunk 1]
```diff
@@ -44,6 +44,16 @@ $ cd /path/to/codeigniter
 $ bin/server.sh
 ```
 
+#### Arguments
+
+You can give arguments like CodeIgniter4 serve command.
```
Comment (kenjis) [inline README.md]: This is BC break.
See the current usage: <https://github.com/kenjis/codeigniter-composer-installer/pull/34>

Why do we need to break the compatiblity?

Comment (ytetsuro) [reply README.md]: > Why do we need to break the compatiblity?

CodeIgniter4 not compatible CodeIgniter3.

Do I not need to support document root?
(CodeIgniter4 serve can not change the document root)

Comment (kenjis) [reply README.md]: > CodeIgniter4 not compatible CodeIgniter3.

It is not enough reasoning to break the compatibility.
The command is from this project.

> Do I not need to support document root?

Yes. I have no reasons to break.

Comment (ytetsuro) [reply README.md]: fixed!

**Pull request closed without merging**